### PR TITLE
Fix stress histogram test

### PR DESCRIPTION
**Pull request by amadio**
Replace direct floating point comparison with `AreEqualRel()` comparison with a tolerance.

### CODE REVIEW

**Comment by amadio** [comment]
My clang-format gives different results than Travis...

File: test/stressHistogram.cxx [hunk 1]
```diff
@@ -7030,44 +7030,33 @@ int findBin(ROOT::Fit::BinData& bd, const double *x)
...
 
       double value2 = 0, error2 = 0;
       const double *x2 = bd2.GetPoint(bin, value2, error2);
 
-      equals &= ( value1 == value2 );
-      equals &= ( error1 == error2 );
-      for ( unsigned int j = 0; j < ndim; ++j )
-      {
-         equals &= fabs(x1[j] - x2[j]) < 1E-15;
+      if (value1 != value2) return false;
```
Comment (pcanal) [inline test/stressHistogram.cxx]: Why keep this equality but replaced the one for error[12]?  Aren't both floating points?

**Comment by pcanal** [comment]
> My clang-format gives different results than Travis...
Are you using clang-format 3.9 (like Travis) or 4.0?

File: test/stressHistogram.cxx (hunk 1 above)
Comment (amadio) [inline test/stressHistogram.cxx]: Good point. I though value was an integer from the output of my debug statements, but looking at the interface it is indeed floating point. I will update it.

**Comment by amadio** [comment]
I'm using version 4.0.

**Comment by pcanal** [comment]
How is the result different?  If the file that Travis saw was clang-formatted with $ROOTSYS/.clang-format and clang-format 4.0 then there is a regression in clang-format that we need to understand.

**Comment by amadio** [comment]
Yes, there is probably a bug, or the default configuration for something we don't set manually changed.
I usually just do a git clang-format with each commit hash in my PR as argument and apply all changes until I see no more suggested changes.

**Comment by pcanal** [comment]
>  or the default configuration for something we don't set manually changed.

We tried to set everything explicit.  If you find out what it is, we ought to update the .clang-format.

**Comment by amadio** [comment]
Ok, for now I just manually applied the changes that clang-format wants.

**Comment by dpiparo** [comment]
Thanks @amadio : merged!

**Pull request merged**